www/source/stylesheets/_demo.scss

```diff
@@ -2,6 +2,7 @@
 
 #demo-step {
   margin-bottom: rem-calc(200);
+  z-index: 0;
```

Comment:
This looks okay, but I'm wondering why`.demo--page` has a z-index of 100 then, since they seem to be applied alongside one another (on the same element). Should we just remove that setting of 100 instead of applying 0 to the id selector? I guess if we're using `.demo--page` on elements that aren't also `#demo-step` it might make sense...